### PR TITLE
Adds Lunan flag sprites

### PR DESCRIPTION
**Pull request by listerla**
Added a Lunan flag, flagpatch, and banner, as well as replacing the accent tag.

Sprites are by Rose (StevIII) and were a part of [this](https://forums.aurorastation.org/topic/18906-a-flag-for-luna/) lore application, which was accepted. 

Tested ingame and working.

### CODE REVIEW

**Comment by listerla** [comment]
!review

**Comment by GMR25** [comment]
Replacing the accent tag here feels like replacing the accent tag of the Earth with Sol’s flag. One is clearly so much more iconic and identifiable to the average person, that it feels like a downgrade.

**Comment by SleepyGemmy** [comment]
> Replacing the accent tag here feels like replacing the accent tag of the Earth with Sol’s flag. One is clearly so much more iconic and identifiable to the average person, that it feels like a downgrade.

i think its fine. luna and earth didn't have a flag and now that luna has a flag it makes some sense to make the accent tag use that flag, as with other accent tags.

**Review by SleepyGemmy — APPROVED** [review]

**Comment by GMR25** [comment]
> i think its fine. luna and earth didn't have a flag and now that luna has a flag it makes some sense to make the accent tag use that flag, as with other accent tags.

The moon feels markedly more identifiable at a glance and visually interesting to me. Not all accent tags are flags so far, looking at Xanu where it’s a(n extremely small) scenic view that captures the beauty of the world.

**Comment by SleepyGemmy** [comment]
> > i think its fine. luna and earth didn't have a flag and now that luna has a flag it makes some sense to make the accent tag use that flag, as with other accent tags.
> 
> The moon feels markedly more identifiable at a glance and visually interesting to me. Not all accent tags are flags so far, looking at Xanu where it’s a(n extremely small) scenic view that captures the beauty of the world.

probably best to ask the lore team what they think is best.

**Comment by listerla** [comment]
Old is preferred, including by me. Will revert.

File: html/changelogs/lunanflag.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: Pirouette, StevIII
+
+delete-after: True
+
+changes:
+  - imageadd: "Added a Lunan banner, flag, and flagpatch to loadout."
+  - imageadd: "Replaced the Lunan accent sprite with one inspired by the flag."
```
Comment (FluffyGhoster) [inline html/changelogs/lunanflag.yml:7]: If the accent sprite is not being changed anymore, this line of the changelog have to go

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

File: html/changelogs/lunanflag.yml (hunk 1 above)
Comment (listerla) [reply html/changelogs/lunanflag.yml]: Fixed.

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**